docker_start.sh

```diff
@@ -3,4 +3,6 @@
 
 PORT=3000
```

Comment:
This value of 3000 matches (duplicates) the value set for `container_port` in the `locals.tf` file. 

We think this is fine, as it's being done in all the Node apps, but just curious if this is 100% correct @tobafolaranmi-ch ? Maybe there are plans to tidy this up during the fire-break?

Reply:
It varies in different services, so it depends on what env var the application is looking for